src/config/cfg_rules.ini

```diff
@@ -11,7 +11,7 @@ section = ifp
 section = secrets
 section = kcm
 section_re = ^secrets/users/[0-9]\+$
-section_re = ^domain/.*$
+section_re = ^domain/[0-9A-Za-z.]\+$
```

Comment:
I think we should not allow "\" or "@" in domain name because we use them as default separator in fq names.

We might also use blacklist instead of whitelist.